### DISCUSSION TITLE
Query by custom field value

### DISCUSSION THREAD

**Discussion by ShahriarKh** [wp-graphql/wp-graphql#2208] [Advanced Custom Fields]
I've made a custom post type called `discount` & it has two fields called `amount` and `code` - made with ACF.
I want to query a discount by passing its code as an argument like this:
```graphql
query MyQuery {
  discounts(where(code: "1234abcd")) {
    details {
      amount
    }
  }
}
```
or
```graphql
query MyQuery {
  discountBy(code: "1234abcd") {
    details {
      amount
    }
  }
}
```
However, `discountBy` only accepts `discountId`, `id`, `slug` & `uri`.
How can I make it accept a custom field value?

**Comment by Fire4All** [comment]
Hello Sir,

have figuered out how to do this?
I am facng th same problem...

Kind regards,
Ben

**Reply by ShahriarKh** [reply]
Nope. Not yet.

**Comment by TonyTheJet** [comment]
Have you had a look at this plugin? https://www.wpgraphql.com/extenstion-plugins/wpgraphql-meta-query/

I think it may offer that ability.
